src/cq_cam/command.py

```diff
@@ -124,34 +115,22 @@
 from cq_cam.visualize import cached_occ_color
 
 
-class CommandVector(ABC):
+class CommandVector:
     __slots__ = ("x", "y", "z")
 
     def __init__(self, x=None, y=None, z=None):
         self.x = x
         self.y = y
         self.z = z
 
-    @abstractmethod
-    def to_vector(self, origin: cq.Vector, relative=False) -> cq.Vector:
-        pass
-
-
-class RelativeCV(CommandVector):
-    def to_vector(self, origin: cq.Vector, relative=False):
-        warnings.warn("Relative CV is deprecated", DeprecationWarning)
-        if relative:
-            x = 0 if self.x is None else self.x
-            y = 0 if self.y is None else self.y
-            z = 0 if self.z is None else self.z
-        else:
-            x = origin.x + self.x if self.x else origin.x
-            y = origin.y + self.y if self.y else origin.y
-            z = origin.z + self.z if self.z else origin.z
-        return cq.Vector(x, y, z)
+    def __eq__(self, __value: CommandVector) -> bool:
+        if self.x == __value.x and self.y == __value.y and self.z == __value.z:
+            return True
+        return False
```

Comment:
Few points..

1) By python convention the 2nd arg to `__eq__` is typically called `other`
2) Double underscore has a special purpose in python ([name mangling](https://docs.python.org/3/tutorial/classes.html#private-variables))
3) "if x then return True else return False" syntax often is just a verbose form of "return x" or "return bool(x)"
4) `__eq__` should generally support comparing against anything (so type `other: Any` and not throw AttributeError if other is not a CommandVector (but rather return [NotImplemented](https://docs.python.org/3/library/constants.html#NotImplemented))

So for example..

```suggestion
    def __eq__(self, other) -> bool:
        try:
            return self.x == other.x and self.y == other.y and self.z == other.z
        except AttributeError:
            return NotImplemented
```

Reply:
Good to know! Done!